pkg/sync/v16/data/Deployment.apps/openshift-azure-logging/omsagent-rs.yaml

```diff
@@ -88,7 +93,19 @@ spec:
       nodeSelector:
         beta.kubernetes.io/os: linux
         node-role.kubernetes.io/compute: "true"
+        kubernetes.io/role: agent
```

Comment:
None of our compute nodes have this label, causing this to fail scheduling,

Reply:
> None of our compute nodes have this label, causing this to fail scheduling,

> @ehashman,  there shouldnt be change to the labels since  agent in ARO v3 already has the labels which are working, Hi, @olga-mir , can point to the existing labels.